lib/hanami/cli/commands/application/server.rb

```diff
@@ -41,8 +44,8 @@ def initialize(server: Hanami::CLI::Server.new)
           end
 
           # @api private
-          def call(...)
-            server.call(...)
+          def call(port: DEFAULT_PORT, **kwargs)
```

Comment:
@jodosha, do we need it? We're already setting the default in the `option` definition.